syntax = "proto3";

message Page {
    string url = 1;
    int32 network_type = 2;
    int32 device_speed = 3;
    bytes manifest = 4;
}

message Policy {
    string policy = 1;
}

service PolicyService {
    rpc GetPolicy(Page) returns (Policy);
}
